papermill/tests/test_api.py

```diff
@@ -36,6 +36,10 @@ def test(self):
         )
         assert_frame_equal(nb.dataframe, expected_df)
 
+        with self.assertRaises(PapermillException) as excinfo:
```

Comment:
`pytest.raises` rather than self.assertRaises?

Reply:
To be fair, this is a unittest object for self.Raises is perfectly valid.